ecosystem/nft-metadata-crawler-parser/src/utils/constants.rs

```diff
@@ -11,3 +11,12 @@ pub const MAX_JSON_REQUEST_RETRY_SECONDS: u64 = 30;
 
 /// Allocate 90 seconds for downloading large image files
 pub const MAX_IMAGE_REQUEST_RETRY_SECONDS: u64 = 90;
+
+/// Default maximum file size for files to be downloaded
+pub const DEFAULT_MAX_FILE_SIZE_BYTES: u32 = 5_000_000;
```

Comment:
This is just a default ofc but in prod we should probs set this to like 10-15mb instead.